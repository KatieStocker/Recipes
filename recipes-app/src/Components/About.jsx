
const About = () => {

    return (
        <>
            <h1 className="about-title">About Us</h1>
            <div className="about-text">
                <p>Straight to the Recipe was created as a project based on a problem I, and I am sure many other recipe followers, encounter when you just want to find a recipe and not have to scroll through paragraphs and paragraphs of text about the creators life-story.</p>
                <p>This website aims to provide a better presentation of recipes to users, skipping the life-story entirely and going straight to the point by keeping each section simple to find and easy to browse through.</p>
            </div>
        </>
    )
}
export default About;