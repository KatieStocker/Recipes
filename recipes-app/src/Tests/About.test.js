import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import About from '../Components/About.jsx'

describe(`Client tests - About test suite`, () => {

    test('it should match the snapshot', () => {
        expect(render(<Router><About /></Router>)).toMatchSnapshot();
    });
});