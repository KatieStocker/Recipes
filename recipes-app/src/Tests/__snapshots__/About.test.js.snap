// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Client tests - About test suite it should match the snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1
        class="mb-3"
      >
        About Us
      </h1>
      <div
        class="about-text"
      >
        <p>
          Straight to the Recipe was created as a project based on a problem I, and I am sure many other recipe followers, encounter when you just want to find a recipe and not have to scroll through paragraphs and paragraphs of text about the creators life-story.
        </p>
        <p>
          This website aims to provide a better presentation of recipes to users, skipping the life-story entirely and going straight to the point by keeping each section simple to find and easy to browse through.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <h1
      class="mb-3"
    >
      About Us
    </h1>
    <div
      class="about-text"
    >
      <p>
        Straight to the Recipe was created as a project based on a problem I, and I am sure many other recipe followers, encounter when you just want to find a recipe and not have to scroll through paragraphs and paragraphs of text about the creators life-story.
      </p>
      <p>
        This website aims to provide a better presentation of recipes to users, skipping the life-story entirely and going straight to the point by keeping each section simple to find and easy to browse through.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
